### PR TITLE
Issue #11446: Update EmptyLineSeparatorCheckTest to use verifyWithInlineConfigParser

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Resolves update for EmptyLineSeparatorCheckTest in https://github.com/checkstyle/checkstyle/issues/11446.

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java [hunk 1]
```diff
@@ -186,9 +186,8 @@ public void testJavadocCommentAfterPackageWithImports() throws Exception {
         final String[] expected = {
             "2:1: " + getCheckMessage(MSG_SHOULD_BE_SEPARATED, "/*"),
         };
-        verify(checkConfig,
-                getPath("InputEmptyLineSeparatorJavadocCommentAfterPackage.java"),
-                expected);
+        execute(checkConfig,
+                getPath("InputEmptyLineSeparatorJavadocCommentAfterPackage.java"));
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java:190]: Remove "expected" variable

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java (hunk 1 above)
Comment (Rahulkhinchi03) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java:190]: Changes done

Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java:188]: You need to convert to new method only places where we expect exception.

All other should be converted to method that use inlined config in Input files.

Comment (Rahulkhinchi03) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java:188]: I believe that we should not use `execute `here as I am getting PMD error and If I am using `veriftWithInlineConfigParser` and changing the `config `in input files then all tests are passing.

But after changing the input files, I am getting code coverage issue.

Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java:188]: update test to not use "verify"  and use verifyWithInlineConfigParser

**Pull request closed without merging**